src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -871,24 +871,24 @@ When you are [building a custom decision](#build-decision), you can use our topo
 * Given access to topology feature by a New Relic representative
 * Limit of 10K vertices per New Relic organization per environment
 
-### Topology setup
+### How does topology correlation work? [#topology-explained]
 
-Here's an overview of what needs to be done to set up topology correlation:
+![New Relic topology explained 4](./images/topology-4.png "topology-4.png")
+<figcaption>In this service map, the hosts and apps are the vertices, and the lines showing their relationships are the edges.</figcaption>
 
-1. A topology graph has been created using the NerdGraph aiTopologyCollector APIs.
-2. Vertices with "defining attributes" exist in the topology and they have been connected together with edges to indicate relationship.
-3. The defining attributes on the vertices are also present on your incident events within Incident Intelligence.
+Topology correlation relies on two main concepts:
 
-### How does topology correlation work? [#topology-explained]
+* **Vertices**: Vertices is the plural of vertex. A vertex is a monitored entity and the source from which your incident events are coming from, or describing the problematic symptom about. A vertex has attributes (key/value pairs) configured for it, like entity GUIDs or other IDs, which allow it be associated with incoming incident events.
```

Comment:
I don't think you need to explain that vertices is the plural of vertex. I do prefer defining the singular term though.

Reply:
I want to do something with the end of this sentence... "A vertex is a monitored entity and the source from which your incident events are coming from, or describing the problematic symptom about."

How about breaking it up? "A vertex is a monitored entity. It's the source of your problematic symptoms, or what we call incident events.